packages/evm/src/index.ts

```diff
@@ -58,7 +58,7 @@ export {
   RustBN254,
   validateEOF,
 }
-
+export { JSONifyStepTrace } from './opcodes/util.ts'
```

Comment:
Suggestion to rename this to `StepTraceJSON`. Why is this exported?

Reply:
We use it in t8n and that's in `vm` 🙃

Reply:
Yer its fine, sorry for some reason I thought it was a type and not a method, and AFAIK you dont have to explicitly export types in order to be picked up by the IDE